cookiecutter.yaml

```diff
@@ -2,8 +2,9 @@ repo_directory: html
 name: ICESat-2 Hackweek
 about:
     description:
-        "We are in the early stages of organizing a fifth event as part of this series, and we need your help! Consider joining the ICESat-2 hackweek organizing team. We will partner with you to design tutorials and projects, and provide a variety of professional development opportunities along the way. Click on the button below to learn more:"
-    learn_more: https://escience.notion.site/UW-Hackweek-Organizing-Team-Application-b7e5742a4c7e4cad8b4ece80ee171ab0
+        "We are now accepting applications for hackweek participants."
+        # "We are in the early stages of organizing a fifth event as part of this series, and we need your help! Consider joining the ICESat-2 hackweek organizing team. We will partner with you to design tutorials and projects, and provide a variety of professional development opportunities along the way. Click on the button below to learn more:"
+    learn_more: http://event.hackweek.io/ # https://escience.notion.site/UW-Hackweek-Organizing-Team-Application-b7e5742a4c7e4cad8b4ece80ee171ab0
```

Comment:
Good catch! Turns out `event.hackweek.io` defaults to http instead of https (at least from the [GitHub page](https://github.com/uwhackweek/event-pages)).

Reply:
There is a [checkbox](https://github.com/uwhackweek/event-pages/settings/pages), which I thought I checked at the time of setup. It is now though ...